packages/perseus/src/widgets/numeric-input/score-numeric-input.ts

```diff
@@ -111,7 +111,7 @@ function scoreNumericInput(
 
     const normalizedAnswerExpected = rubric.answers
         .filter((answer) => answer.status === "correct")
-        .every((answer) => Math.abs(answer.value) <= 1);
+        .every((answer) => answer.value == null || Math.abs(answer.value) <= 1);
```

Comment:
I had to change this in order to convince TypeScript that we're handling answers with no `value`, which exist in our content.

...though now that I'm reading the code again, I think I did it wrong. `Math.abs(null)` is 0, but `Math.abs(undefined)` is NaN, and NaN is not `<= 1`. So the existing behavior is to set `normalizedAnswerExpected` to `false` if any `answer.value` is undefined, but to `true` if every `answer.value` is either null or at most 1. My change allows `normalizedAnswerExpected` to be true when both `null` and `undefined` values are present.

I'm not sure how much this detail matters given that the current logic is extremely weird already (see the characterization tests above) but I don't want to break existing exercises.